client/my-sites/plugins/categories/use-categories.tsx

```diff
@@ -1,4 +1,4 @@
-import { useI18n } from '@wordpress/react-i18n';
+import { __ } from '@wordpress/i18n';
```

Comment:
Needed to use this `__` in order to be able to translate the categories in the prefetch middleware (before the provider and components are initialized).